orderer/common/channelparticipation/validator_test.go

```diff
@@ -39,16 +39,6 @@ func TestValidateJoinBlock(t *testing.T) {
 			expectedChannelID: "my-channel",
 			expectedErr:       nil,
 		},
-		{
-			testName: "Invalid block data hash",
-			joinBlock: func() *cb.Block {
-				b := proto.Clone(validJoinBlock).(*cb.Block)
-				b.Header.DataHash = []byte("bogus")
-				return b
-			}(),
-			expectedChannelID: "",
-			expectedErr:       errors.New("invalid block: Header.DataHash is different from Hash(block.Data)"),
-		},
```

Comment:
Same question as above, I think this use case should be tested and protected against.

Reply:
done